src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -209,11 +209,14 @@
    allow-header-cert-info :- schema/Bool
    oid-map :- acl/OIDMap]
   (if-let [cert (request->cert request allow-header-cert-info)]
-    (let [extensions (ssl-utils/get-extensions cert)
+    (let [oid-map (merge acl/default-oid-map oid-map)
```

Comment:
Since there are already a couple of places where the `default-oid-map` is getting merged with the custom `oid-map`, I wonder if it would be better to merge them in one place (at a higher-level) and pass the merged map on down.  Maybe in `ring-middleware/authorization-check?`